content/news/2025/08/04/pybamm-25.8.md

```diff
@@ -0,0 +1,40 @@
+---
+title: PyBaMM 25.8 has been released!
+summary: PyBaMM version 25.8 has now been released with several new features and improvements.
+date: 2025-08-04
+shortcutDepth: 1
+---
+
+PyBaMM 25.8 has now been released! We would like to thank all the [contributors](https://pybamm.org/teams/) who made this release possible.
+
+The full list of changes can be found in the [CHANGELOG](https://pybamm.org/changelog/) file, but here we provide a deeper overview of the main features of this release.
+
+## Python 3.9 support removed
+
+Support for Python 3.9 has been removed in this release. The supported Python versions are now 3.10 to 3.12.
+
+## SPM + 3D thermal model
+_Implemented by [Rishab Kumar Jha](https://github.com/Rishab87)_
+
+This version brings several features that allow for 2D and 3D simulations in PyBaMM. The main features of this release are the 3D mesh support for prismatic and cylindrical geometries, plotting functionality for 2D&3D variables and, more importantly the `Basic3DThermalSPM`, which couples a 3D geometry (pouch or cylindrical) for the thermal problem with the Single Particle Model for the electrochemical problem.
+
+For more details, check the [example scripts for 3D models](https://github.com/pybamm-team/PyBaMM/tree/develop/examples/scripts/3d_examples). 2D and 3D functionality is ongoing work and will be further developed in the upcoming releases.
```

Comment:
```suggestion
For more details, check the [example scripts for 3D models](https://github.com/pybamm-team/PyBaMM/tree/c06726d2bf7e32ccf7a896dea7077332786b99db/examples/scripts/3d_examples). 2D and 3D functionality is ongoing work and will be further developed in the upcoming releases.
```

Just changing the link here to a permanent one so that it does not drift if we were to reorganise these files again